lib/order.rb

```diff
@@ -0,0 +1,20 @@
+
+class Order
+  attr_reader :current_order
+
+  def initialize(menu)
+    @current_order = []
+  end
+
+  def select_item(item)
+    if item == 'chips' or item == 'sausage' or item =='fish'
```

Comment:
Style/AndOr: Use `||` instead of `or`. (https://github.com/bbatsov/ruby-style-guide#no-and-or-or)